Note Save Listing

dtimmon1 opened the pull request:
User can now see currently and or previously saved/submitted notes to the server in list form on the Webapp GUI.

Steps:

1. Load webapp.

2. Fill out the Title and Content boxes with information. 

3. Hit the save/submit button once finished.

3. Once your you've created and saved/submitted your content, refresh the page. Underneath the body of the content box, you should see your save (title, content, date created, and last modified) added to the web UI. If there are saves already logged, you should see those as well.

dtimmon1 committed:
Note Save Listing

User can now see currently and or previously saved/submitted notes to the server in list form on the Webapp GUI.

sduman commented on main.py:
```diff
@@ -4,8 +4,33 @@
+
 THIS_FILE_PATH = os.path.dirname(os.path.realpath(__file__))
 
+saveData = [{
```
Since this is test data and you didn't have any piece of the python section you should remove the changes for this file.

sduman commented on templates/index.html:
```diff
@@ -27,6 +27,10 @@ <h1>QuickNotes</h1>
 
+    <ul id="noteList">
+    </ul>
+
```
Remove excess spacing here.

dtimmon1 committed:
main.py and index.hmtl resolution 120419

-main.py
Contained an old variable for data retrieval testing purposes. Removed lines 12-33.

-index.html
Contained an excess space between the id tags for noteList and servertime. Removed line 33.

dtimmon1 committed:
Update for html.index - noteList

Collapsed (removed excess spacing) from noteList unordered list as requested.

sduman commented on main.py:
```diff
@@ -60,5 +63,26 @@ def submitNote():
         return (qnTitle + '\n' + qnContent+ dateMessage)
 
-    return app
+    @app.route('/fileread', methods=['GET'])
```
These are Scott's changes I believe, but since this wasn't part of your ticket you should just go ahead and discard all of your tracked changes to this whole file. You might have to do something like revert the branch or delete the PR and resubmit but since you weren't making any changes to the server code, we ought not to include an extra revision if we don't need to.
